packages/web-react/src/components/Grid/__tests__/Grid.test.tsx

```diff
@@ -106,4 +98,44 @@ describe('Grid', () => {
     expect(element).not.toHaveStyle({ '--grid-spacing-x-desktop': 'var(--spirit-space-1200)' });
     expect(element).not.toHaveStyle({ '--grid-spacing-y-desktop': 'var(--spirit-space-1200)' });
   });
+
+  it('should render with horizontal alignment', () => {
+    render(<Grid alignmentX="center" data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXCenter');
+  });
+
+  it('should render with default alignment classes', () => {
+    render(<Grid data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXStretch Grid--alignmentYStretch');
+  });
+
+  it('should render with horizontal and vertical alignment', () => {
+    render(<Grid alignmentX="center" alignmentY="center" data-testid={testId} />);
+
+    expect(screen.getByTestId(testId)).toHaveClass('Grid--alignmentXCenter Grid--alignmentYCenter');
+  });
+
+  it('should render with responsive horizontal alignment', () => {
+    render(<Grid alignmentX={{ mobile: 'left', tablet: 'center', desktop: 'right' }} data-testid={testId} />);
```

Comment:
@crishpeen mentioned (and I second that) there should be a test case with only some breakpoints. For example:

```suggestion
    render(<Grid alignmentX={{ tablet: 'center', desktop: 'right' }} data-testid={testId} />);
```

I think you can adjust this case and don't need to add another one.